src/components/ex-table/ex-table.scss

```diff
@@ -0,0 +1,24 @@
+table {
+  @apply mt-component-spacing-small inline-block max-w-full overflow-x-scroll;
+
+  * {
+    @apply border border-black border-solid;
+  }
+
+  th {
+    @apply bg-primary text-left text-white;
+  }
+
+  td,
+  th {
+    @apply px-3 py-1;
+  }
+
+  tr:nth-child(2n) {
+    background-color: #f5f8fa;
+  }
+
+  caption {
+    @apply border-0;
```

Comment:
Caption atop of a table should be font-weight bold.

Reply:
caption is bold now